package.json

```diff
@@ -21,8 +21,14 @@
   "eslintConfig": {
     "extends": [
       "react-app",
-      "react-app/jest"
-    ]
+      "react-app/jest",
+      "eslint:recommended",
+      "plugin:react/recommended",
```

Comment:
An extension to ESLint which can perform numerous functions (? I believe this is the answer?)